test/functionalTest/cases/3272_custom_notification_sending_0-length_request/custom_notification_sending_0-length_request.test

```diff
@@ -0,0 +1,423 @@
+# Copyright 2020 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Custom notification sending 0-length request
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. Create E1/T1
+# 02. Create custom subscription with payload="", includePayload=false
+# 03. Create custom subscription with payload="", includePayload=true
+# 04. Create custom subscription without payload, includePayload=false
+# 05. Create custom subscription without paylaod, includePayload=true
+# 06. Create custom subscription with payload="{ %22x%22: 1}", includePayload=false
+# 07. Create custom subscription with paylaod="{ %22x%22: 1}", includePayload=true
+# 08. Dump accumulator to see the notifications
+#
```

Comment:
I think it would be a good idea to add:

```
# 09. GET /v2/subscriptions to check subscriptions payload
```

to check that the payload of the subscriptions is as expected.

Reply:
Updated